### PR TITLE
31: Improve the submit page

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: src/app/pages/submission-pages/data-page/data-page.component.ts [hunk 1]
```diff
@@ -44,14 +44,15 @@ export class DataPageComponent implements OnInit {
...
-      this.files['_embedded']['files'].forEach(element => {
-        element.status_label = this.formatFileStatus(element.status);
-      });
-    });
-
+    this.fileService.getActiveSubmissionsFiles(this.activeSubmission).subscribe(
+      (data) => {
+        this.files = data;
+        // format the file status string and store it as status_label for display
+        this.files['_embedded']['files'].forEach(element => {
```
Comment (meladawy) [inline src/app/pages/submission-pages/data-page/data-page.component.ts]: I do suggest that we remove these 3 lines and from the template we do something like 

{{formatFileStatus(file.status)}}

Comment (ke4) [reply src/app/pages/submission-pages/data-page/data-page.component.ts]: OK, I will do that, but I have not changed these lines.
I just moved the service method to the relevant service and use it from there.

Comment (ke4) [reply src/app/pages/submission-pages/data-page/data-page.component.ts]: I fixed it

Comment (meladawy) [reply src/app/pages/submission-pages/data-page/data-page.component.ts]: Yea I see, I just suggested a better implementation for this.

**Pull request merged**